src/common/components/file-issue-details-button.tsx

```diff
@@ -55,7 +55,13 @@ export class FileIssueDetailsButton extends React.Component<FileIssueDetailsButt
         const body = this.props.deps.issueDetailsTextGenerator.buildGithubText(data);
 
         const encodedIssue = `/new?title=${encodeURIComponent(title)}&body=${encodeURIComponent(body)}`;
-        return `${this.props.issueTrackerPath}${encodedIssue}`;
+        let baseUrl = this.props.issueTrackerPath;
+
+        if (baseUrl.indexOf('/issues') === -1) {
```

Comment:
Hmm, I think this will not work correctly for URLs like https://github.com/issuesender/project or https://github.com/magazine/issues.

Reply:
The second one works... but I see your point. This is actually just a quick fix as the intention is to remove the old experience as soon as we validate the new one